src/zeroband/training/world.py

```diff
@@ -5,7 +5,7 @@
 import zeroband.training.envs as envs
 
 
-class WorldInfo:
+class World:
```

Comment:
```suggestion
class WorldInfo:
```

not sure about just World

Reply:
brevity is nice, `world.rank` etc. make sense imo?